Bixal/move.mil
# 167926322/remove ppm min max

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- As a Move.mil User, I need the Personally Procured Move (PPM) Estimator to calculate my incentive cost accurately so I have an estimate for how much the government will pay me for my PPM.
- Removes min-max ranges and just add the single estimate

## Testing

To verify the changes proposed in this pull request…

1. Make a calculation on PPM estimator tool
1. Verify there are not ranges anymore

## Screenshots

![Screen Shot 2019-08-20 at 1 20 50 PM](https://user-images.githubusercontent.com/37001835/63369164-5fe12700-c34d-11e9-9cf4-7eb85420b893.png)
